examples/quantization_w4a16/llama3_example.py

```diff
@@ -67,7 +67,7 @@ def tokenize(sample):
 print("========== SAMPLE GENERATION ==============")
 dispatch_for_generation(model)
 sample = tokenizer("Hello my name is", return_tensors="pt")
-sample = {key: value.to("cuda") for key, value in sample.items()}
+sample = {key: value.to(model.device) for key, value in sample.items()}
```

Comment:
we do this in several examples, could you update all of those? I can update also on this branch